port/win/port_win.h

```diff
@@ -336,6 +336,12 @@ extern int GetMaxOpenFiles();
 std::string utf16_to_utf8(const std::wstring& utf16);
 std::wstring utf8_to_utf16(const std::string& utf8);
 
+using ThreadId = int;
+
+extern void SetCpuPriority(ThreadId id, int priority);
```

Comment:
`priority` is not platform independent enough, so it is not clear what it means for Windows and other platform. This is probably another reason we should consider to only use priority 19 for now on Linux and avoid this `priority` argument.

Reply:
I originally created a function called `SetLowestCpuPriority` which sets the priority to `19`. But afterwards, I find that the user needs to update priority for each backup, so I changed to the current signature. I think the user's feature request is specifically for Linux, I doubt whether we want to make it portable.

Reply:
created `CpuPriority` enum to be platform neutral